pkg/controller/static-resource-controller/service_test.go

```diff
@@ -153,7 +178,13 @@ func TestGetSpireServerService(t *testing.T) {
 	assert.Equal(t, "Service", svc.Kind)
 	assert.Equal(t, "zero-trust-workload-identity-manager", svc.Namespace)
 
-	expectedLabels := requiredServerResourceLabels
+	expectedLabels := map[string]string{
+		"app.kubernetes.io/name":       "server",
```

Comment:
The labels needs an update as 
```
    app.kubernetes.io/name: spire-server
    app.kubernetes.io/instance: <cr-name>-<namespace>
    app.kubernetes.io/part-of: zero-trust-workload-identity
    app.kubernetes.io/component: control-plane
    app.kubernetes.io/managed-by: ztwim-operator
```
The component names that can be set as operator, csi, control-plane, node-agent, discovery